features/step_definitions/question_steps.rb

```diff
@@ -59,20 +68,24 @@
   end
   fill_in 'How long will the session be? (optional)', with: '5 minutes'
   fill_in 'What do participants need to bring? (optional)', with: 'Nothing'
+  Percy::Capybara.snapshot(page, name: 'Time')
   click_button 'Continue'
 
   fill_in 'If you allow travel expenses, what is the maximum allowed?', with: '50.00'
   fill_in 'If you allow food expenses, what is the maximum allowed?', with: '20.00'
+  Percy::Capybara.snapshot(page, name: 'Expenses')
   click_button 'Continue'
 
   choose 'Yes'
 
   choose 'Cash'
   fill_in 'Incentive value', with: '10.50'
-
+  Percy::Capybara.snapshot(page, name: 'Incentive')
```

Comment:
Would we be ok to use Symbols for this, just so that it's obvious that they mean the same thing as the symbols in `ResearchSession::Steps::PARAMS`?

Reply:
??? Not sure. Isn't this ok? how would I get the symbols into it... should you Even use symbols. What do you think @joel?

Reply:
I mean just use `name: :incentive` instead of `name: 'Incentive'` 😄

Reply:
(then it's easier mentally to make the connection with the [steps](https://github.com/barnardos/consent-form-builder-rails/blob/24f470d2336b2c969760a429db9233e209d85fa4/app/models/research_session/steps.rb#L8-L18), that's all)